### DISCUSSION TITLE
Using Wild Cards on Unicode, ASCII, and Latin characters

### DISCUSSION THREAD

**Discussion by Vanir-Labs** [ImageMagick/ImageMagick#7027] [Help] [answered]
### ImageMagick version

7.1.1-26

### Operating system, version and so on

Windows 10

### Description

Hello! I'm trying to run a script to change a bunch of files with symbols. 
My symbols reflect this "A_A£AII’„AI«§Añ1€O§_ø­15" even this website is tunrning some into boxes "".
What I need to do is get this sharens cript to accept wild cards using /u and cls. here's my sample.

`mogrify -sharpen 0x.3 "A_A£AII’„AI«§Añ1€O§_ø­15.bmp"`

If we can turn the .bmp into a wild card.bmp that is what I am trying to do.

**Comment by urban-warrior** [comment]
We need your image filename, uncorrupted.  Put your command in a Unicode-capable file and attach it here.  We'll download it and investigate the issue further.

**Reply by Vanir-Labs** [reply]
Thanks for your time. I appreciate it. 
Here's is the docx file with some name examples listed.
[example.docx](https://github.com/ImageMagick/ImageMagick/files/13928595/example.docx)

**Comment by urban-warrior** [comment, marked as answer]
Your command worked fine for us, that is, ImageMagick properly read and rewrote the Unicode filename.  We're using ImageMagick 7.1.1-26, Windows 11, and we used `magick mogrify` rather than `mogrify`.  We're not sure if its a Windows 10 issue, we do not have that OS to test against.

**Reply by Vanir-Labs** [reply]
Thanks for testing thsi for me. 
Is it possible that Notepad++, the editor I'm using to make the batch file, is what is causing the issue?
I ran it through the CMD directly and it worked as Iw anted it to. Perhaps it's because I had it in a batch file before.

Thank you so much for your time helping me.
